src/app.js

```diff
@@ -13,6 +13,7 @@ const path = require('path')
 const i18next = require('i18next')
 const Backend = require('i18next-fs-backend')
 const middleware = require('i18next-http-middleware')
+const { InternalCache, RedisHelper } = require('elevate-node-cache')
```

Comment:
@ankitpws what is the use of including in app.js?

Reply:
@rakeshSgr fixed